.travis.yml

```diff
@@ -31,10 +31,10 @@ script: tools/travis-test.sh
 
 matrix:
     include:
-        - otp_release: 20.0
+        - stage: test
+          otp_release: 20.0
           env: PRESET=dialyzer MIX_ENV=test
-
-script: tools/travis-test.sh
-
-after_success:
-  - tools/travis-docker.sh
+        - stage: deploy
+          elixir: 1.5.1
```

Comment:
what's the reason to use 1.5.1 version?

most users will install latest version:
elixir/unknown,now 1.5.2-1 all [installed]

Reply:
Well, technically Elixir is used here only in order to run mix. Deploy stage builds docker image which uses internally different version of Elixir (listed in dockerfiles). This version is actually a lot older (1.4.5) simply because it hasn't got updated for a while. Anyway, this may only affect performance (if there were any performance upgrades between 1.4.5 -> 1.5.x) since as I said - this is an docker container - Erlang/Elixir in it are only driving MongoosePush and doesn't affect any other environments.